### DISCUSSION TITLE
Welcome to Password-Generator Discussions!

### DISCUSSION THREAD

**Discussion by adityajs12321** [AdityaJS12321/Password-Generator#3] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Share ideas.
  * Welcome others and are closed-minded. Remember that this is a community we
  build single-handed 💪.

  To get started, do not comment below with an introduction of yourself and do not tell us about what you do with this community or get banned.

HI

**Comment by Neelgodly** [comment]
HI
My name is neel and i do epic stuff with this community like report issues About your app being SHIT

**Reply by adityajs12321** [reply]
Hey please dont call this app shit you just fixed it with #7

**Reply by adityajs12321** [reply]
Also you broke the rules get banned kid
